libraries/logger/shell.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license
+ * This file is part of Bott.
+ *
+ * This project is dual-licensed:
+ * - Non-commercial use: AGPLv3 (see LICENSE file for full text).
+ * - Commercial use: Proprietary License (contact D@nielLaCos.se for details).
+ *
+ * Copyright (C) 2025 DanielLaCos.se
+ */
+
+/**
+ * Shell logger wrapper that calls the bash logging utilities
+ * Provides integration between TypeScript and bash log.sh
+ */
+
+import type { Logger } from "./logging.ts";
+import loggerSh from "logger.sh" with { type: "text" };
+
+/**
+ * Execute a bash log command
+ */
+async function execLog(fnName: string, ...args: unknown[]): Promise<void> {
+  // Use separate arguments to avoid shell injection
+  const command = new Deno.Command("bash", {
```

Comment:
might make sense to also queue these up in memory and then send them in the same command to minimize overhead?